typify-impl/tests/new_type_with_enum.json

```diff
@@ -0,0 +1,14 @@
+{
+  "$id": "https://example.com/arrays.schema.json",
+  "$schema": "https://json-schema.org/draft/2020-12/schema",
+  "title": "Everything",
+  "type": "object",
+  "properties": {
+    "enum_property": {
+      "description": "name",
+      "type": "string",
+      "maxLength": 32,
```

Comment:
This is a case where I think we should simply ignore `maxLength` or use it to cull the list of value `enum` values.